module1.js

```diff
@@ -52,8 +60,12 @@ function question3(number) {
   the output would be: [2,4,6]
 */
 
-function question4(array) {
-  // TODO:
+function question4(a) {
+  var array = [];
+  for (var i = 0; i < a.length; i += 1) {
```

Comment:
Are you sure this passes the test now? `i += 1` is the same as `i++` and that means you add every single element in the input array to the one you're returning. The task is to add every second element, so you need to increment `i` by 2 :)